LoopKitTests/QuantityFormatterTests.swift

```diff
@@ -6,230 +6,230 @@
+        XCTAssertEqual("7.8\(String.nonBreakingSpace)mmol\(String.wordJoiner)/\(String.wordJoiner)L", formatter.string(from: LoopQuantity(unit: .millimolesPerLiter, doubleValue: 7.84))!)
+        XCTAssertEqual("12.0\(String.nonBreakingSpace)mmol\(String.wordJoiner)/\(String.wordJoiner)L", formatter.string(from: LoopQuantity(unit: .millimolesPerLiter, doubleValue: 12))!)
         formatter.unitStyle = .short
         XCTAssertEqual("mmol\(String.wordJoiner)/\(String.wordJoiner)L", formatter.localizedUnitStringWithPlurality())
-        XCTAssertEqual("6.0\(String.wordJoiner)mmol\(String.wordJoiner)/\(String.wordJoiner)L", formatter.string(from: HKQuantity(unit: HKUnit.millimolesPerLiter, doubleValue: 6))!)
-        XCTAssertEqual("7.8\(String.wordJoiner)mmol\(String.wordJoiner)/\(String.wordJoiner)L", formatter.string(from: HKQuantity(unit: HKUnit.millimolesPerLiter, doubleValue: 7.8))!)
+        XCTAssertEqual("6.0\(String.wordJoiner)mmol\(String.wordJoiner)/\(String.wordJoiner)L", formatter.string(from: LoopQuantity(unit: .millimolesPerLiter, doubleValue: 6))!)
+        XCTAssertEqual("7.8\(String.wordJoiner)mmol\(String.wordJoiner)/\(String.wordJoiner)L", formatter.string(from: LoopQuantity(unit: .millimolesPerLiter, doubleValue: 7.8))!)
         formatter.unitStyle = .long
         XCTAssertEqual("millimoles per liter", formatter.localizedUnitStringWithPlurality())
-        XCTAssertEqual("5.5 millimoles per liter", formatter.string(from: HKQuantity(unit: HKUnit.millimolesPerLiter, doubleValue: 5.5))!)
-        XCTAssertEqual("0.0 millimoles per liter", formatter.string(from: HKQuantity(unit: HKUnit.millimolesPerLiter, doubleValue: 0))!)
-        XCTAssertEqual("1.0 millimoles per liter", formatter.string(from: HKQuantity(unit: HKUnit.millimolesPerLiter, doubleValue: 1))!)
+        XCTAssertEqual("5.5 millimoles per liter", formatter.string(from: LoopQuantity(unit: .millimolesPerLiter, doubleValue: 5.5))!)
+        XCTAssertEqual("0.0 millimoles per liter", formatter.string(from: LoopQuantity(unit: .millimolesPerLiter, doubleValue: 0))!)
+        XCTAssertEqual("1.0 millimoles per liter", formatter.string(from: LoopQuantity(unit: .millimolesPerLiter, doubleValue: 1))!)
     }
 
     func testUnitRounding() {
         let value = 1.2345
-        var unit: HKUnit = .milligramsPerDeciliter
+        var unit: LoopUnit = .milligramsPerDeciliter
         XCTAssertEqual(unit.roundForPreferredDigits(value: value), 1)
 
-        unit = unit.unitDivided(by: .internationalUnit())
+        unit = .milligramsPerDeciliterPerInternationalUnit
         XCTAssertEqual(unit.roundForPreferredDigits(value: value), 1)
 
-        unit = unit.unitDivided(by: .minute())
-        XCTAssertEqual(unit.roundForPreferredDigits(value: value), 1)
+        unit = .milligramsPerDeciliterPerMinute
+        XCTAssertEqual(unit.roundForPreferredDigits(value: value), 1.2)
```

Comment:
@ps2 this is that test I believe was incorrect before this change. Not sure why it was passing as the declaration has mg/dL/min with 1 rounded digit instead of 0.